src/models/hooks/eventReportPilot.js

```diff
@@ -183,6 +182,58 @@ const updateGoalText = async (sequelize, instance, options) => {
   );
 };
 
+const createOrUpdateNationalCenterUserCacheTable = async (sequelize, instance, options) => {
+  try {
+    const { ownerId, collaboratorIds } = instance;
+    const userIds = [ownerId, ...collaboratorIds].filter((f) => f);
```

Comment:
can owner be a collab?

Reply:
nvm prob doesn't matter how you are using it.

Reply:
yes. we don't need to dedupe though, as we just use them in a query: user.id in (userIds)